components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/java/org/wso2/carbon/user/mgt/ui/UserAdminClient.java

```diff
@@ -230,10 +230,10 @@ public void setRoleUIPermission(String roleName, String[] rawResources) throws A
         }
     }
 
-    public void bulkImportUsers(String fileName, DataHandler handler, String defaultPassword)
+    public void bulkImportUsers(String userStoreDomain, String fileName, DataHandler handler, String defaultPassword)
             throws AxisFault {
         try {
-            stub.bulkImportUsers(fileName, handler, defaultPassword);
+            stub.bulkImportUsers(userStoreDomain, fileName, handler, defaultPassword);
         } catch (Exception e) {
             handleException(e);
         }
```

Comment:
Need to apply same changes to https://github.com/wso2/product-is/blob/master/modules/samples/identity-mgt/info-recovery-sample/src/main/java/org/wso2/sample/inforecovery/client/UserAdminClient.java